src/gov/nist/core/LexerCore.java

```diff
@@ -342,6 +382,15 @@ public boolean startsId() {
             return false;
         }
     }
+    
+    public boolean startsIp() {
+        try {
+            char nextChar = lookAhead(0);
+            return isIPV6addressChar(nextChar);
```

Comment:
Because I have already checked the very first char is '[' or not at 
`NameValue nameValue(char separator)` function in ParserCore class
So I remove this function.